### PR TITLE
COMMERCE-12613

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-12613

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceAccelerators.path [hunk 1]
```diff
@@ -340,6 +340,11 @@
 	<td>//div[contains(@class,'price')]//span[(@class='price-on-application price-value')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRODUCT_EDIT_BUNDLED_PRICE_ON_APPLICATION</td>
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceAccelerators.path:344]:
```suggestion
	<td>MINI_CART_EDIT_ITEM_PRICE_ON_APPLICATION</td>
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditBundledProducts.testcase [hunk 2]
```diff
@@ -516,16 +522,14 @@ definition {
...
-				var i = ${i} + 1;
-			}
+			//var i = 1;
+			//for (var discount : list "10,20,0,0") {
+			//	AssertTextEquals.assertPartialText(
+			//		key_index = ${i},
+			//		locator1 = "CommerceAccelerators#MINI_CART_DISCOUNT_LEVELS",
+			//		value1 = ${discount});
+			//	var i = ${i} + 1;
+			//}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditBundledProducts.testcase:532]: @Brunno0liveira check if the test passes with the discount levels locally then uncomment these lines. Thanks!

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
The issues are not related to the changes PR can be forward.

@victoroscuro

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**